README.md

```diff
@@ -103,13 +103,19 @@ where
 (this is basically the output of the MLC method)
 - ``true_labels`` is a comma-separated list of true labels (ground truth)
 
-Note that you can omit the ``true_labels`` section if you only want to predict the 
+Note that you can omit the ``true_labels`` section if you only want to predict the
```

Comment:
suggestion: "if you only want to obtain a prediction of the quality of the MLC result."